### DISCUSSION TITLE
posts/practica-javascript-w3schools/

### DISCUSSION THREAD

**Discussion by giscus** [marcosruiz/marcosruiz.github.io#5] [General]
# posts/practica-javascript-w3schools/

Objetivo y duración

https://marcosruiz.github.io/posts/practica-javascript-w3schools/

**Comment by rtapiamateos** [comment]
![Hola](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTXkz-bmTcuXVDsh19ifigUqauRMlF4ll8RFQ&usqp=CAU)

**Comment by Guxilu** [comment]
![](https://i.redd.it/svfc2l89zkk51.jpg)

**Comment by marcosruiz** [comment]
![Learning JavaScript](https://i.redd.it/k0tdut0w1gm81.jpg)

**Comment by marcosruiz** [comment]
![Not my type](https://i.redd.it/vorgttfjn6l51.jpg)
